### ISSUE TITLE
buggy Uridium 2 (jpn) playback

### ISSUE THREAD

**Issue by ciros88** [aargirakis/BZRPlayer#400]
jpn.LOADMUS: last two subtunes playback consists of 3 seconds silence
jpn.MUSIC plays flawlessly
jpn.MUSIC2:  last subtune playback consists of 3 seconds silence
[Uridium2.zip](https://github.com/aargirakis/BZRPlayer/files/11309140/Uridium2.zip)

**Comment by aargirakis** [comment]
Exactly same behavior in XM Play on Windows and EaglePlayer 2.01 on Amiga

**Comment by ciros88** [comment]
So, what do u think about? They are taken from here: https://www.exotica.org.uk/wiki/Uridium_2

**Comment by aargirakis** [comment]
Yes, I tried that one too. Well, do you know any other player that might play those subsongs? Otherwise is has to be a bad rip.
